msteams-platform/graph-api/rsc/rsc-overview.md

```diff
@@ -0,0 +1,178 @@
+---
```

Comment:
Please retain the name of this file as resource-specific-consent.md so that existing shortcuts to the page continue to work.

Reply:
Renamed .md file as suggested.